engine/src/main/java/org/hibernate/search/backend/impl/lucene/ExclusiveIndexWorkspaceImpl.java

```diff
@@ -27,7 +28,8 @@ public ExclusiveIndexWorkspaceImpl(DirectoryBasedIndexManager indexManager, Work
 		boolean async = ! BackendFactory.isConfiguredAsSync( cfg );
 		if ( async ) {
 			int commitInterval = PropertiesParseHelper.extractFlushInterval( indexManager.getIndexName(), cfg );
-			commitPolicy = new ScheduledCommitPolicy( writerHolder, indexManager.getIndexName(), commitInterval );
+			ErrorHandler errorHandler = context.getErrorHandler();
+			commitPolicy = new ScheduledCommitPolicy( writerHolder, indexManager.getIndexName(), commitInterval, errorHandler );
 		}
 		else {
 			commitPolicy = new PerChangeSetCommitPolicy( writerHolder );
```

Comment:
It's not needed for that one as it's not asynchronous. The problem we're fixing is that an exception would kill the timer thread, and never commit again.